Makefile

```diff
@@ -8,7 +8,7 @@ OKD_VERSION ?= 4.12.0-0.okd-2023-02-18-033438
 MICROSHIFT_VERSION ?= 4.12.5
 BUNDLE_EXTENSION = crcbundle
 CRC_VERSION = 2.15.0
-COMMIT_SHA?=$(shell git rev-parse --short HEAD)
+COMMIT_SHA?=$(shell git rev-parse --short=6 HEAD)
```

Comment:
As per man page of `git-rev-parse` looks like this can be vary for different host, so I think the binary created in different container and then test execution happend in different container.
```
--short[=length]
           Same as --verify but shortens the object name to a unique prefix with at least length characters. The minimum length is 4, the default is the effective value of the core.abbrev configuration variable (see git-config(1)).
```

Reply:
Also in the commit log mention why this change is required for future reference.

Reply:
> this can be vary for different host

Yes, that's why I made it fixed lenght. But there may be a better solution, e.g. adjust git settings on the hosts directly and leave this at default short? I don't know... 

Anyways, I rebased it as it was and included reasoning in the commit log. Should be OK now.